src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests/AssertionExtensions.Android.cs

```diff
@@ -45,12 +47,20 @@ public static AColor ColorAtPoint(this Bitmap bitmap, int x, int y, bool include
 
 		public static Bitmap ToBitmap(this AView view)
 		{
+			var layout = new FrameLayout(view.Context);
+			layout.LayoutParameters = new FrameLayout.LayoutParams(500, 500);
```

Comment:
Why 500 as height and width?

Reply:
just randomly picked something 
No science to this really. 

It's just the container frame and then the added control will just calculates its own height/width

If we get to a point where we're screen shotting largers views we need to take pictures of we can feed these properties in